packages/embark-contracts-manager/src/index.js

```diff
@@ -123,7 +123,8 @@ class ContractsManager {
 
           self.events.request("blockchain:contract:create", {abi: contract.abiDefinition, address: contract.deployedAddress}, async (contractObj) => {
             try {
-              const value = req.body.value !== undefined ? web3.utils.toWei(req.body.value, 'ether') : 0;
+              let value = typeof req.body.value === "number" ? req.body.value.toString() : req.body.value;
+              value = web3.utils.toWei(value, 'ether');
```

Comment:
Are all values that might returned from a contract method call/send appropriate to pass through `toWei`?